src/pages/components/CertificateCo2Post.js

```diff
@@ -111,9 +117,21 @@ export default function CertificateCo2Post() {
     handleSubmitStep,
   ])
 
+  const certificateDates = useMemo(() => {
+    return (
+      dataCertFound?.events.reduce(
+        (acc, { event, occurred_at }) => {
+          acc[event] = formatDate(occurred_at)
+          return acc
+        },
+        { initiated: undefined, issued: undefined }
```

Comment:
Would it makes sense to add `revoked: undefined` or do you think we should add this later ( especially since there is a ticket for that )